<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserDialog</class>
 <widget class="QDialog" name="CUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;RID</string>
          </property>
          <property name="buddy">
           <cstring>editRID</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editRID">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&amp;Username</string>
          </property>
          <property name="buddy">
           <cstring>editUsername</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editUsername">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&amp;Full name</string>
          </property>
          <property name="buddy">
           <cstring>editFullname</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editFullname">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&amp;Comment</string>
          </property>
          <property name="buddy">
           <cstring>editComment</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="editComment">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;Home directory</string>
          </property>
          <property name="buddy">
           <cstring>editHomeDir</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="editHomeDir">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&amp;Member of these groups</string>
         </property>
         <property name="buddy">
          <cstring>listGroups</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listGroups"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupACB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Account bits</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbACB1">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbACB2">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cbACB3">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbACB4">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbACB5">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cbACB6">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cbACB7">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cbACB8">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="cbACB9">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbACB10">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cbACB11">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="cbACB12">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbACB13">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cbACB14">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="cbACB15">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLoginCount">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnClearPassword">
       <property name="text">
        <string>Clear password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnlock">
       <property name="toolTip">
        <string>Lock, unlock and enables user account</string>
       </property>
       <property name="text">
        <string>Unlock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPromote">
       <property name="toolTip">
        <string>Make user an administrator</string>
       </property>
       <property name="text">
        <string>Promote</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddToGroup">
       <property name="text">
        <string>Add to group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveFromGroup">
       <property name="text">
        <string>Remove from group</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editRID</tabstop>
  <tabstop>editUsername</tabstop>
  <tabstop>editFullname</tabstop>
  <tabstop>editComment</tabstop>
  <tabstop>editHomeDir</tabstop>
  <tabstop>listGroups</tabstop>
  <tabstop>cbACB1</tabstop>
  <tabstop>cbACB4</tabstop>
  <tabstop>cbACB7</tabstop>
  <tabstop>cbACB10</tabstop>
  <tabstop>cbACB13</tabstop>
  <tabstop>cbACB2</tabstop>
  <tabstop>cbACB5</tabstop>
  <tabstop>cbACB8</tabstop>
  <tabstop>cbACB11</tabstop>
  <tabstop>cbACB14</tabstop>
  <tabstop>cbACB3</tabstop>
  <tabstop>cbACB6</tabstop>
  <tabstop>cbACB9</tabstop>
  <tabstop>cbACB12</tabstop>
  <tabstop>cbACB15</tabstop>
  <tabstop>btnClearPassword</tabstop>
  <tabstop>btnUnlock</tabstop>
  <tabstop>btnPromote</tabstop>
  <tabstop>btnAddToGroup</tabstop>
  <tabstop>btnRemoveFromGroup</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>CUserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
